packages/camel-catalog/assembly/src/main/resources/schemas/PipeConfiguration.json

```
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "description": "Schema for Pipe configuration",
+  "properties": {
+    "name": {
+      "title": "Name",
+      "description": "Name of the Pipe",
+      "type": "string"
+    },
+    "labels": {
+      "additionalProperties": {
+        "default": "",
+        "type": "string"
+      },
+      "title": "Additional Labels",
```

Since these are the only labels in the form, could we please rename the title to `Labels` instead?
```suggestion
      "title": "Labels",
```